llama/model.py

```diff
@@ -92,46 +96,65 @@ def __init__(self, args: ModelArgs, world_size: Optional[int] = None,
 
         init_method = lambda x: x
 
-        self.wq = ColumnParallelLinear(
-            args.dim,
-            args.n_heads * self.head_dim,
-            bias=False,
-            gather_output=False,
-            init_method=init_method,
-            world_size=world_size,
```

Comment:
These arguments are lost.

Reply:
Missed during rebase, let me update. Thanks for catching it!